backend/routes/api.php

```diff
@@ -16,3 +16,9 @@
 Route::middleware('auth:api')->get('/user', function (Request $request) {
     return $request->user();
 });
+
+Route::prefix('v1')->group(function () {
+    Route::group(['prefix' => 'auth', 'namespace' => 'Api\\Auth'], function () {
+        Route::put('/update', 'AuthController@update');
```

Comment:
it is incorrect name of route, REST doesn't have verbs in the resource name. For update users route should be

PUT /api/v1/users